lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1203,20 +1209,27 @@ def cmd_route_tabs(str, words)
     ret
   end
 
+  # Print save help information
   def cmd_save_help
-    print_line "Usage: save"
-    print_line
-    print_line "Save the active datastore contents to disk for automatic use across restarts of the console"
-    print_line
-    print_line "The configuration is stored in #{Msf::Config.config_file}"
+    print_line 'Usage: save [options]'
     print_line
+    print_line 'Save the active datastore contents to disk for automatic use across restarts of the console'
+    print @@save_opts.usage
   end
 
-  #
-  # Saves the active datastore contents to disk for automatic use across
-  # restarts of the console.
-  #
-  def cmd_save(*args)
+  # Delete all saved options for modules from the config file.
+  # No framework options will be deleted.
+  def cmd_save_delete_all_modules_configs
```

Comment:
We can inline this directly to avoid the need for a method here, or move it to the protected section, and remove teh `cmd_` prefix